Crashlytics/Crashlytics/Controllers/FIRCLSRolloutsPersistenceManager.m

```diff
@@ -57,18 +64,20 @@ - (void)updateRolloutsStateToPersistenceWithRollouts:(NSData *_Nonnull)rollouts
 
   NSFileHandle *rolloutsFile = [NSFileHandle fileHandleForUpdatingAtPath:rolloutsPath];
 
-  dispatch_async(FIRCLSGetLoggingQueue(), ^{
-    @try {
-      [rolloutsFile seekToEndOfFile];
-      NSMutableData *rolloutsWithNewLineData = [rollouts mutableCopy];
-      [rolloutsWithNewLineData appendData:[@"\n" dataUsingEncoding:NSUTF8StringEncoding]];
-      [rolloutsFile writeData:rolloutsWithNewLineData];
-      [rolloutsFile closeFile];
-    } @catch (NSException *exception) {
-      FIRCLSDebugLog(@"Failed to write new rollouts. Exception name: %s - message: %s",
-                     exception.name, exception.reason);
-    }
-  });
+  if (_rolloutsLoggingQueue) {
```

Comment:
Instead of nesting in an if, could we do a check and log an error if it's invalid?

Eg. 
```
if ( ! _rolloutsLoggingQueue) {
  FIRCLSLogError(...);
}
```